couch2pg/src/importer.js

```diff
@@ -135,12 +156,18 @@ const importChangesBatch = async (couchDb, source) => {
             docsToDelete.length + ' deletions and ' +
             docsToDownload.length + ' new / changed documents');
 
+  console.log('There are approximately ' + pending + ' changes left');
   await loadAndStoreDocs(couchDb, changes.results);
-  await storeSeq(changes.last_seq, source);
+  await storeSeq(changes.last_seq, pending, source);
 
   return changes.results.length;
 };
 
+const getPending = async (couchDb, seq) => {
```

Comment:
In [cht-core](https://github.com/medic/cht-core/blob/5274b92dc25d54680427af8ad55ae7024485113e/api/src/services/monitoring.js#L61) we use request instead of pouch to get the pending docs. Can we explore suing that so that this is more accurate?

Reply:
yea couch-requests seems to be an internal library in cht-core? But yea, maybe we can use axios or whatever plain http thing to get pending instead.
I've added it.

Reply:
Yes, using axios is fine. We should add a test that checks that we set the pending number correctly.

Reply:
added ` it('should save the correct pending count'` and `it('should not throw error when getting pending fails'` to couch2pg unit tests